bucket/sidekick-browser.json

```diff
@@ -0,0 +1,37 @@
+{
+    "version": "104.29.3.25043-e335c0c",
+    "homepage": "https://www.meetsidekick.com/",
+    "description": "A work browser that is focused on productivity, speed, and security.",
+    "license": {
+        "identifier": "Freeware",
+        "url": "https://www.meetsidekick.com/terms-of-use/"
+    },
+    "notes": "Settings are stored in: ~\\AppData\\Local\\Sidekick",

```

Comment:
Is this necessary to show to the user on installation? The notes section is more for listing config files that need to be edited before the program is usable or for more immediate messages

Reply:
Should I delete this section?

Reply:
I'll delete it, there is no use for it.

Reply:
Done.

Reply:
It's good to note, though, if maintainers in the future are wondering whether the app needs to have any config persisted. So do bring it up in future PRs (put it in your PR description)!

Reply:
oh ok good to know.